api/v1/api_v1_snapshots.go

```diff
@@ -12,9 +12,15 @@ import (
 // GetIsiSnapshots queries a list of all snapshots on the cluster
 func GetIsiSnapshots(
 	ctx context.Context,
-	client api.Client) (resp *getIsiSnapshotsResp, err error) {
+	client api.Client, scheduleName string) (resp *getIsiSnapshotsResp, err error) {
 	// PAPI call: GET https://1.2.3.4:8080/platform/1/snapshot/snapshots
-	err = client.Get(ctx, snapshotsPath, "", nil, nil, &resp)
+
+	params := api.NewOrderedValues([][]string{})
```

Comment:
So my only gripe is that this slightly changes the case where `scheduleName` is empty. Instead, of passing `nil` to `params` this change would pass an empty  `OrderedValues` and it's not clear to me that those are equivalent. 

It doesn't help that the tests don't run.

Reply:
I can tell you it works, but I can make the change to just send nil instead of an empty OrderValues